scripts/docker/nginx-oss/apk/Dockerfile

```diff
@@ -12,11 +12,20 @@ COPY $ENTRY_POINT /agent/entrypoint.sh
 RUN set -x \
     && addgroup -g 101 -S nginx \
     && adduser -S -D -H -u 101 -h /var/cache/nginx -s /sbin/nologin -G nginx -g nginx nginx \
-    && apk add \
-            ca-certificates \
-            curl \
-            bash \
-            nginx
+    && apk add ca-certificates \
+               curl \
+               openssl \
+               bash \
+               nginx
+
+# Setup nginx agent repository
+RUN curl -o /etc/apk/keys/nginx_signing.rsa.pub https://nginx.org/keys/nginx_signing.rsa.pub
+
+RUN printf "%s%s%s%s\n" \
+    "@nginx-agent " \
```

Comment:
I don't think the `@nginx-agent` string is required here. See my other PR here https://github.com/nginx/agent/pull/353/files#diff-96cc175e545424410f6b4f903f43bc3679bfec94219d0d7ec069a53b8715d22aR230